_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation-filter-popin/smart-shopping-campaign-creation-popin-dimension.vue

```diff
@@ -81,7 +80,19 @@ export default {
   computed: {
     dimensionChosen: {
       get() {
-        return this.$store.state.smartShoppingCampaigns.dimensionChosen;
+        let final = this.$store.state.smartShoppingCampaigns.dimensionChosen;
```

Comment:
Prefer something like `dimensionChosen` instead of `final` to make it easier to understand in the future